client/src/transmitter/index.ts

```diff
@@ -1,33 +1,73 @@
-import isIp from 'is-ip'
-
 import { copyToClipboard } from '../clipboard'
 import cast from './cast'
+import isLocalOrIp from './util/isLocalOrIp'
+
+const { body } = document
+
+const _video = document.getElementById('video') as HTMLVideoElement
 
-const isLocalHost = (hostname) => {
-  return hostname === 'localhost' || hostname.endsWith('.localhost')
+const _canvas = document.getElementById('canvas') as HTMLCanvasElement
+
+function resizeCanvas() {
+  const { innerWidth, innerHeight } = window
+
+  _canvas.width = innerWidth
+  _canvas.height = innerHeight
 }
 
-const { body } = document
+window.addEventListener('resize', () => {
+  resizeCanvas()
+})
```

Comment:
**nit:** 
```suggestion
window.addEventListener('resize', resizeCanvas)
```

Reply:
If I made that reduction, it would be hard to put a breakpoint on the `"resize"` event.